extension/readme/src/views/Report/report.js

```diff
@@ -57,8 +57,8 @@ class Report extends Component {
                 type: "report",
                 id: "123",
                 tags: [],
-                fake_st: "null",
-                rating_st: "null",
+                fake_st: "true",
```

Comment:
So the default reliability status of an article is true until proven otherwise?

Reply:
no, it is a deprecated field, set to true to avoid validation